.github/workflows/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM ghcr.io/kokkos/ci-containers/fedora:latest
+CMD /bin/bash
+
+WORKDIR /home/kokkos/src/
+COPY kokkos/ /home/kokkos/src/kokkos
+RUN sudo chown -R kokkos:kokkos kokkos
+RUN cmake -S kokkos -B kokkos/build \
+  -DCMAKE_BUILD_TYPE=Release \
+  -DCMAKE_INSTALL_PREFIX=/usr \
+  -DCMAKE_INSTALL_INCLUDEDIR=include/kokkos \
+  -DKokkos_ENABLE_OPENMP=ON \
+  -DBUILD_SHARED_LIBS=ON \
+  -DKokkos_ENABLE_TESTS=ON \
+  -DKokkos_ENABLE_BENCHMARKS=ON \
+  -DKokkos_ENABLE_EXAMPLES=ON \
+  -DKokkos_ENABLE_DEPRECATED_CODE_4=ON \
```

Comment:
My preference here for a "canonical" container would be to have deprecated code disabled

Reply:
Done!